mautic/mautic
# Avoiding exceptions when the value of the choice field is null

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ ]
| New feature/enhancement? (use the a.x branch)      | [ ]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 



#### Description:


When a user saves the "Clicked email" condition with an empty URL field an exception is thrown. This PR fixes it.


#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Create a campaign or use an existing one
3. Add  a "Send Email" action
4. Add a "Clicks email" decision
5. Add a "Limit to Urls contains" field  (or several fields) and left it (them) empty.
6. Try adding. The field is marked as an error field. Before it was 500er.
![image](https://user-images.githubusercontent.com/96085911/220584406-8ec2367d-ff1d-4d1d-83cf-03d3f8ce1674.png)
